### ISSUE TITLE
About pre trained sdf field

### ISSUE THREAD

**Issue by Talegqz** [MoyGcc/vid2avatar#52]
Would you be able to provide the code for obtaining the pre-trained SDF field?
I've noticed that when I modify the SDF network without using the pre-trained field, the loss consistently diverges to NaN.

**Comment by MoyGcc** [comment]
Hi, a similar issue has been raised [here](https://github.com/MoyGcc/vid2avatar/issues/27). You could try to walk around this problem using the tricks that I mentioned in the issue or quickly implement that initialization training following the main idea. I'm sorry that I'm now pushing for a deadline and cannot guarantee that I have a whole block of time for cleaning that part recently.

**Comment by Talegqz** [comment]
Thanks for the clarification. Regarding the pretraining of the SDF, did you only train on the SDF values (1 channel) and disregard the features (256 channels)?

**Comment by MoyGcc** [comment]
Yes, we still keep 257 dims as the network output but only the SDF values are used to compute losses.
